pages/common/chmod.md

```diff
@@ -1,36 +1,24 @@
 # chmod
 
-> Change the access permissions of a file or directory.
+> Change file/directory access permissions.
 > More information: <https://www.gnu.org/software/coreutils/chmod>.
 
-- Give the [u]ser who owns a file the right to e[x]ecute it:
+- Add [r]ead/[w]rite/e[x]ecute permissions (`{{r|w|x}}` is referred later as `{{permissions}}`) for a file/directory owner:
```

Comment:
I think using `{{r|w|x}}` is better for reading. When you define `((permissions))`, people must read the first example's description carefully so that they understand what `((permissions))` stands for, and which also makes people read more lines. It's not the goal of "too long don't read".
```suggestion
- Add [r]ead/[w]rite/e[x]ecute permissions for a file/directory owner:
```

Reply:
I suggest use first cases of placeholders with alternatives not to repeat all cases:

```md
- Add [r]ead/[w]rite/e[x]ecute permissions for a file/directory owner:

`chmod +{{r|w|x}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`

- Add specific permissions for an owner/users in [g]roup/[o]ther users/[a]ll at once:

`chmod {{u|g|o|a}}+{{r}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```

The problem with complete inlining that it's hard to read smth like this:

```md
- Add/remove/replace specific permissions:

`chmod {{u|g|o|a}}{{+|-|=}}{{r|w|x}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```

because several placeholders with alternatives are joined.